orchestration/dashboard.py

```
@@ -0,0 +1,361 @@
+        # Check disk usage for tasks directory
+        try:
+            disk_usage = subprocess.run(
+                ["du", "-sh", self.tasks_dir],
+                check=False,
+                capture_output=True,
+                text=True,
+            )
+            if disk_usage.returncode == 0:
+                size = disk_usage.stdout.split()[0]
+                print(f"   Tasks directory: {size}")
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)